vue/readme.md

```diff
@@ -237,47 +240,53 @@ To preview variation 1 instead of variation 2, the `70` can be updated to any in
 
-To guarantee that all visitors will see only one variation, update the `min` and `max` on each variation. Below, we have increased the likelihood of seeing variation 2 to 100%: 
+To guarantee that all visitors will see only one variation, update the `min` and `max` on each variation in your `variate.json` file. Below, we have increased the likelihood of seeing variation 2 to 100%: 
 
-```json
+```json 
 "variations": [
-                    {
-                        "id": 1,
-                        "traffic_allocation": {
-                            "max": 0,
-                            "min": 0
-                        },
-                        "components": {
-                            "HomeHero": {
-                                "id": 1,
-                                "attributes": {
-                                    "headline": "The A/B testing tool for the modern web"
-                                }
-                            }
-                        }
-                    },
-                    {
-                        "id": 2,
-                        "traffic_allocation": {
-                            "max": 100,
-                            "min": 1
-                        },
-                        "components": {
-                            "HomeHero": {
-                                "id": 1,
-                                "attributes": {
-                                    "headline": "The A/B testing tool that marketers and developers can agree on"
-                                }
-                            }
-                        }
-                    }
-                ]
+    {
+        "id": 1,
+        "traffic_allocation": {
+            "max": 0,
+            "min": 0
+        },
+        "components": {
+            "HomeHero": {
+                "id": 1,
+                "attributes": {
+                    "headline": "The A/B testing tool for the modern web"
+                }
+            }
+        }
+    },
+    {
+        "id": 2,
+        "traffic_allocation": {
+            "max": 100,
+            "min": 1
+        },
+        "components": {
+            "HomeHero": {
+                "id": 1,
+                "attributes": {
+                    "headline": "The A/B testing tool that marketers and developers can agree on"
+                }
+            }
+        }
+    }
+]
 ```
-Now, all visitors will be bucketed into the same variation: 
+Now, after reloading, all visitors will be bucketed into the same variation: 
 
 <img :src="$withBase('/variate-vue-force-variation-config.png')" alt="The same variation shows regardless of the experiment bucket">
 
 <br>
-<img :src="$withBase('/variate-vue-experiment-bucket-1.png')" alt="Experiment bucket in local storage">
\ No newline at end of file
+<img :src="$withBase('/variate-vue-experiment-bucket-1.png')" alt="he same variation shows regardless of the experiment bucket">
+
+::: warning Overriding Targeting Conditions
```

Comment:
@alhankeser  Shall we add another disclaimer here, telling them to be careful not to commit the config with a 100% traffic allocation, unless it's what they want?

Reply:
Nothing should be done unless it's what someone wants to do. We could put that disclaimer on basically every sentence of the docs otherwise :) We won't need this portion of the docs once there is built-in preview capability and regardless, most teams like to set variations to 100% for QA purposes. It's pretty common.